lib/api.ts

```diff
@@ -210,9 +195,12 @@ export class Api extends EventEmitter {
     if (!scriptUrl) {
       scriptUrl = routes.childcontrollerScript
     }
-    const createItemHeaders = await this.retrieveCreateItemHeaders(scriptUrl)    
-    const response = await this.fetch('createItem', url, createItemHeaders)
-
+    const createItemHeaders = await this.retrieveCreateItemHeaders()
+    const response = await this.fetch('createItem', url, {
+      method: 'POST',
+      ...createItemHeaders,
```

Comment:
Just for the sake of it, would it make sense to exclude a host header, if one where present, in the fetched config?